mathesar_ui/src/components/dropdown/Dropdown.svelte

```diff
@@ -33,8 +33,12 @@
     }
   }
 </script>
+<<<<<<< HEAD
```

Comment:
I assume while merging, there were some manual changes leading to git's conflict markers to be included.
Please remove these conflict markers, and merge the changes in the Button components manually.

If this is your first time handling merge conflicts, I recommend going through [Resolving a merge conflict using the command line](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line). You could also check with the team on Matrix, they might have better recommendations for reading materials/videos.